dark_emulator/darkemu/de_interface.py

```diff
@@ -986,6 +986,47 @@ def get_bias(self, logdens, redshift):
         """
         return self.g1.get_bias(redshift, logdens)
 
+    def get_bias_massthreshold(self, Mth, redshift):
+        """get_bias_massthreshold
+
+        Compute the linear bias, :math:`b(>M_\mathrm{th})`, for a mass threshold halo sample.
+
+
+        Args:
+            Mth (float): Halo mass threshold in :math:`[h^{-1}M_\odot]`
+            redshift (float): redshift at which the lens halos are located
+
+        Returns:
+            float: linear bias factor
+        """
+        Mp = M * 1.01
+        Mm = M * 0.99
+        logdensp = np.log10(self.mass_to_dens(Mp, redshift))
+        logdensm = np.log10(self.mass_to_dens(Mm, redshift))
+        bp = self.get_bias(logdensp, redshift)
+        bm = self.get_bias(logdensm, redshift)
+        return (bm * 10**logdensm - bp * 10**logdensp) / (10**logdensm - 10**logdensp)
```

Comment:
This function looks like the one to compute bias at a point mass. Shouldn't it be simply like
```python
def get_bias_massthreshold(self, Mth, redshift):
    logdens = np.log10(self.mass_to_dens(Mth, redshift))
    return self.get_bias(logdens, redshift)
```
?

Reply:
Oops. I just copy-and-pasted from get_bias_mass, and forgot to modify the actual content... Thanks!